kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -787,4 +787,20 @@
     });
   }
 
+  var draftbutton = $('.btn-draft');
+  var url = $('.btn-draft' ).data( 'draft-url' );
+  var draftmessage = $('.draft-message').html();
+  draftbutton[0].addEventListener('click', function() {
+    var both_data = $('#both_form').serializeArray();
+    var doc_data = $('#doc_form').serializeArray();
+    var rev_data = $('#rev_form').serializeArray();
```

Comment:
Nit: camelCase variable names rather than snake_casing. 

Also, can we think of some more meaningful ID names for these form elements? At first glance, it's really hard for me to decipher what `both_form` `doc_form` and `rev_form` mean. If you can help clarify their differences for me a bit I can try to think of some more informative IDs.